release/test/regenerate_releases_json_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
+source ../_releases_json.sh
+
+function _assertEquals {
+    if [ "${1}" = 1 ] || [ "${1}" = true ]
+    then
+        echo "Test ${FUNCNAME[1]} passed"
+    else
+        echo "Test ${FUNCNAME[1]} failed"
+
+        if [ -n "${2}" ]
+        then
+            echo "${2}"
+        fi
+    fi
+}
+
+function test_merge_json_snipets {
+    local earliest_url
+    local latest_url
+
+    earliest_url="$(jq -r '.[0].url' < "$(find ./20*dxp*.json | head -n 1)")"
+    latest_url="$(jq -r '.[0].url' < "$(find ./20*dxp*.json | tail -n 1)")"
```

Comment:
@lucasmiranda0, can you please declare the variables and assign their content at once?